core/unit_test/tools/TestEventCorrectness.hpp

```diff
@@ -450,13 +455,20 @@ TEST(kokkosp, parallel_scan_no_fence_view) {
         << "skipping since the OpenMPTarget backend has unexpected fences";
 #endif
 
+  // Execute the parallel_scan first without looking for fence events.
+  // Depending on the backend implementation and the order of tests,
+  // it might be that the first call to parallel_scan is reallocating scratch
+  // memory which implies a fence when deallocating. We are not interested in
+  // detecting this event.
+  TestScanFunctor tf;
+  Kokkos::View<typename TestScanFunctor::value_type> v("scan_result");
+  Kokkos::parallel_scan("dogs", Kokkos::RangePolicy<>(0, 1), tf, v);
+
```

Comment:
Again, dogs are the best people, but maybe a more informative label in anticipating of a future failure event 😄